server/endpoints/api/system/index.js

```diff
@@ -206,6 +206,69 @@ function apiSystemEndpoints(app) {
       }
     }
   );
+  app.delete(
+    "/v1/system/remove-documents",
+    [validApiKey],
+    async (request, response) => {
+      /*
+      #swagger.tags = ['System']
```

Comment:
@yond1994 Can you update this tag to `System Settings` instead of `System`?

Otherwise g2g!

Reply:
resolved